index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
In this file you use 2 space for tabs in the others you use 4. I think you should use one or the other.

Reply:
The are all 2.